front/components/vaults/VaultSideBarMenu.tsx

```diff
@@ -511,7 +516,7 @@ const VaultDataSourceViewSubMenu = ({
     >
       {isExpanded && (
         <Tree isLoading={isVaultDataSourceViewsLoading}>
-          {vaultDataSourceViews.map((ds) => (
+          {sortedViews.map((ds) => (
```

Comment:
We usually sort by "importance". (there is a function for that)

Reply:
@Fraggle what's the name of that function? 🤔

Reply:
orderDatasourceByImportance and co.

Reply:
Looked up and not really useful in the case of websites and folders as it doesn't sort alphabetically